packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -170,11 +170,21 @@ export async function meterResponse(
       logger.info({ res }, 'Response sent')
     })
     .catch((err) => {
-      logger.error(ErrorMessage.UNKNOWN_ERROR)
-      logger.error(err)
+      logger.error({ err }, 'Caught error in outer endpoint handler')
```

Comment:
I could imagine us wanting to set up monitoring and alerting for this. Could you create an ErrorMessage so we can have log based metrics here?